versioned/persist/tests/src/main/java/org/projectnessie/versioned/persist/tests/extension/DatabaseAdapterExtension.java

```diff
@@ -230,9 +230,9 @@ private Object resolve(
     }
 
     Object assign;
-    if (type.isAssignableFrom(DatabaseAdapter.class)) {
+    if (DatabaseAdapter.class.isAssignableFrom(type)) {
       assign = databaseAdapter;
```

Comment:
not sure how it works :thinking: We're returning `databaseAdapter`, which would be assigned to a variable of type `type`, right? In this case the old comparison was more correct, I think.

Did you mean `type.isAssignableFrom(databaseAdapter.getClass())`?

Reply:
This one is to distinguish `DatabaseAdapter.class` from `VersionStore.class` - the other one is below.

Reply:
Sorry, I was confused, please ignore my comment.

Reply:
`isAssignableFrom` IS confusing